web/src/styles/onboarding.css

```
@@ -2179,3 +2179,306 @@
+.ceo-card-actions {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+}
+
+/* Inline spinner (submitting state only — never full-card animation) */
+.ceo-card-spinner {
+  display: inline-block;
+  width: 12px;
+  height: 12px;
+  border: 2px solid currentColor;
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)